build.gradle

```diff
@@ -217,8 +217,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
     implementation 'javax.servlet:jstl:1.2'
 
-    implementation 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.10.0'
-    implementation 'org.elasticsearch:elasticsearch:7.10.0'
+    implementation('org.elasticsearch.client:elasticsearch-rest-high-level-client:7.10.0')
```

Comment:
Is the elasticsearch plugin not used? Could we instead bump the version to the latest? 
```
implementation 'org.elasticsearch:elasticsearch:8.9.1'
implementation 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.17.12'
```

Also nit: Can you remove the parenthesis around the reference? 
`implementation 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.10.0'`

Reply:
The elasticsearch plugin is actually dependency of `elasticsearch-rest-high-level-client`. Also just bumping the version of `elasticsearch-rest-high-level-client` didn't help as long as elasticsearch was pinned to 7.10.0. I'll leave the version as-is since I don't think we're using the search endpoints anymore.